### DISCUSSION TITLE
Cutting for strong bonding of large parts

### DISCUSSION THREAD

**Discussion by Traktorist3d** [OrcaSlicer/OrcaSlicer#6362] [Ideas]
Hi, if it is feasible, please add cutting of parts with automatic addition of fasteners for "tongue and groove" for datals that can only be printed horizontally and they are large. An example is on the screenshot, I manually cut the part in Solidworks, if such a tool was in OrcaSlicer, it would be great and would save a lot of time.

![image](https://github.com/user-attachments/assets/d8d08b95-f3dd-40bf-ad0e-f9bf4c3bace8)
![image](https://github.com/user-attachments/assets/f608c204-08d0-4811-9f1d-eb121371d7af)
![image](https://github.com/user-attachments/assets/455c68f0-1efe-4c92-9a74-aa0dc0fb8153)
![image](https://github.com/user-attachments/assets/66475572-76fd-4b4c-9d92-30004f2a9fda)

**Comment by Avpman2** [comment]
@Traktorist3d Can I impose on you to cut up a model for me? I have tried to follow the docs on how to do it but I can't get it right.

**Reply by Traktorist3d** [reply]
I can try, but I cut in solidworks native file and not stl, send the file
